php/Terminus/Models/Organization.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+namespace Terminus\Models;
+
+use stdClass;
+use Terminus\Models\User;
+use Terminus\Models\TerminusModel;
+use Terminus\Models\Collections\OrganizationSiteMemberships;
+//use Terminus\Models\Collections\OrganizationUserMemberships;
+use Terminus\Models\Collections\Workflows;
+
+class Organization extends TerminusModel {
+  protected $site_memberships;
+  protected $user;
+  protected $user_memberships;
+  protected $workflows;
+
+  /**
+   * Object constructor
+   * TODO: Implement OrganizationUserMemberships
+   *
+   * @param [stdClass] $attributes Attributes of this model
+   * @param [array]    $options    Options to set as $this->key
+   * @return [Organization] $this
+   */
+  public function __construct($attributes, $options = array()) {
+    parent::__construct($attributes, $options);
+    if (!isset($this->user)) {
+      $this->user = new User(new stdClass(), array());
+    }
+    $params = array(
+      'organization' => $this,
+      'owner'        => $this,
+      'owner_type'   => 'organization',
+    );
+    $this->site_memberships = new OrganizationSiteMemberships($params);
+    //$this->user_memberships = new OrganizationUserMemberships($params);
+    $this->workflows        = new Workflows($params);
+  }
+
+  /**
+   * Retrieves organization sites
+   *
+   * @return [array] $sites An array of OrganizationSiteMember objects
+   */
+  public function getSites() {
```

Comment:
I think we might want these getters to reference the memberships collection, rather than an array of the contents of the collection. I think we should be clear that it's getting a collection of SiteMemberships which reference sites, but are not the same thing.